app/components/CrashLocationBanner.tsx

```diff
@@ -0,0 +1,21 @@
+import Alert from "react-bootstrap/Alert";
+
+interface CrashLocationBannerProps {
+  privateDriveFlag: boolean | null;
+}
+
+/**
+ * Crash Location Banner is rendered if private flag drive is true or austin full purpose flag is false.
+ */
+export default function CrashLocationBanner({
+  privateDriveFlag,
+}: CrashLocationBannerProps) {
+  return (
+    <Alert variant="warning">
```

Comment:
Like, this just doesn't matter, but there's something awkward to me about how you have to put that bit of whitespace in the interpolated bit of JSX, but I can't think of anything that's clearly better. Putting it inside the quoted conditional string is not great, maybe this? Again, this couldn't be a smaller detail and ignore it pls if you want.

```
<Alert variant="warning">
  {`This crash is not included in Vision Zero statistical reporting because ${
    privateDriveFlag 
      ? "it occurred on a private drive" 
      : "it is located outside of the Austin full purpose jurisdiction"
  }`}
</Alert>
```